caluma/caluma_core/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.conf import settings
+from django.conf.urls import url
+
+from caluma.caluma_user.views import AuthenticationGraphQLView
+
+urlpatterns = [
+    url(
+        r"^graphql",
```

Comment:
I would remove the prefix here so we can add it in the "main" URLs file

Reply:
Done.